cdhweb/people/views.py

```diff
@@ -125,12 +124,11 @@ class StaffListView(ProfileListView):
     show_grant = False
 
     def get_queryset(self):
-        # filter to profiles with staff flag set and exclude postdocs
-        # and students
+        # filter to profiles with staff flag set and exclude students
         # (already ordered by job title sort order and then by last name)
-        return super().get_queryset().staff().not_postdocs().not_students()
+        return super().get_queryset().staff().not_students()
         # NOTE: this won't filter correctly if we ever have someone who
-        # goes from a postdoc or student role to a staff position, however
+        # goes from a student role to a staff position, however
```

Comment:
We have an instance of this now, and I'm curious how the code handles it. Is this comment still accurate, and do we care?

Camey was a PGRA and now is a postdoc. Does she show up as a past student because of the PGRA position (which works independent of the student status) and also show up  as staff? Or does she only show up as staff? (I think either one is probably ok, I'm just curious how it works now. But it would be good to update the comment here if it isn't accurate anymore).

Reply:
Glad you mentioned this, because Camey was actually being excluded from the staff page, so the note was correct. I changed it back so that `not_students()` excludes based only on PU Status and updated the note.

Camey now shows on both the current staff page and the current student page, but I think that's because she's a PM on Shakespeare and Company which has a current grant, not because of her past PGRA role.